src/Simplify.cpp

```diff
@@ -2908,6 +2951,13 @@ void simplify_test() {
     check((x/8)*8 < x - 9, f);
     check((x/8)*8 < x - 7, (x/8)*8 < x + (-7));
 
+    check(min(x, likely(x)), likely(x));
```

Comment:
I wonder if we should add likely calls to the fuzz tester generated expressions?